### PR TITLE
Refactor as a package

### PR DESCRIPTION
**Pull request by kjordahl**
Move everything to an installable package.

Replaces #79.

### CODE REVIEW

**Comment by kjordahl** [comment]
@johntyree Any comment on this? I'd like to merge so I can start reworking the global state on the connections.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distut2ils.core import setup
+
+setup(name='tabs',
+      version='1.0dev',
+      description='TABS vector server',
```
Comment (johntyree) [inline setup.py]: Will the description change when this includes contours?

**Comment by johntyree** [comment]
LGTM, assuming it works.

File: setup.py (hunk 1 above)
Comment (kjordahl) [inline setup.py]: Just made it more generic: "TABS website".

**Pull request merged**